pkgs/development/python-modules/btrfs/default.nix

```diff
@@ -1,22 +1,31 @@
 { lib
 , buildPythonPackage
-, fetchPypi
+, fetchFromGitHub
 }:
 
 buildPythonPackage rec {
   pname = "btrfs";
-  version = "11";
+  version = "12";
 
-  src = fetchPypi {
-    inherit pname version;
-    sha256 = "1w92sj47wy53ygz725xr613k32pk5khi0g9lrpp6img871241hrx";
+  src = fetchFromGitHub {
+    owner = "knorrie";
+    repo = "python-btrfs";
+    rev = "v${version}";
+    sha256 = "sha256-ZQSp+pbHABgBTrCwC2YsUUXAf/StP4ny7MEhBgCRqgE=";
   };
 
+  postInstall = ''
+    mkdir -p "$out/bin"
+    mv bin/btrfs-* "$out/bin"
+  '';
+
+  pythonImportsCheck = [ "btrfs" ];
+
   meta = with lib; {
     description = "Inspect btrfs filesystems";
     homepage = "https://github.com/knorrie/python-btrfs";
     license = licenses.lgpl3Plus;
-    platforms = platforms.linux;
-    maintainers = [ maintainers.evils ];
+    platforms = platforms.linux ++ platforms.darwin;
```

Comment:
there's btrfs on darwin?

Reply:
nvm, just read the initial comment in its entirety

Reply:
```suggestion
```

buildPythonPackage defaults to that.

Reply:
I decided to not change `platforms` for now since btrfs-space-calculator seems to be the only useful thing on darwin, see https://github.com/NixOS/nixpkgs/pull/120638#discussion_r620786877.